app/src/main/java/com/paddlesandbugs/dahdidahdit/brasspound/AbstractMorseInput.java

```diff
@@ -179,6 +184,7 @@ public void run() {
     public void handleKey(KeyEvent event) {
         this.hwPaddle.handleKey(event);
     }
+    public void handleMouse(MotionEvent event) { this.hwPaddle.handleMouse(event); }
```

Comment:
Please insert some  blank lines around the new method and re-format the method wrapped to three lines.